packages/contracts/tsconfig.json

```diff
@@ -3,6 +3,6 @@
   "compilerOptions": {
       "resolveJsonModule": true
   },
-  "include": ["./test"],
+  "include": ["./test", "src/**/*"],
```

Comment:
Why `src/**/*` instead of `./src`? Curious if there is a difference (I'm never quite sure with this field...)

Reply:
not sure but that's how it's defined in the example: https://www.typescriptlang.org/tsconfig#include